client/components/login/LoginButtons/LoginButtons.tsx

```diff
@@ -38,7 +38,7 @@ export default function LoginButtons({
         Înregistrează-te
       </Button>
       <div className="text-right mt-4 text-xs">
-        <Button variant="link" type="button" onClick={onResetPassword}>
+        <Button variant="link" type="button" onClick={onResetPassword} className={`${styles['forgot-password']}`}>
```

Comment:
You don't have to do: 
```
className={`${styles['forgot-password']}`}
```

Since it's just one, you can simply do: `className={styles['forgot-password']}`.